xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/DeletePageConfirmationPage.java

```diff
@@ -67,8 +68,80 @@ public boolean isRecycleBinOptionsDisplayed()
         }
     }
 
+    /**
+     * @return the arrow element used for opening or closing the backlinks panel
+     * @since 14.4RC1
+     */
+    public WebElement getBacklinkPanelExpandButton()
```

Comment:
If this is used only for click, then I'd rather have a "toggle*" or "click*" (high level) method.